client/cloudevents/payload/manifestwork/mainfiest.go

```diff
@@ -0,0 +1,41 @@
+package manifestwork
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"open-cluster-management.io/api/client/cloudevents/types"
+	workv1 "open-cluster-management.io/api/work/v1"
+)
+
+var ManifestEventDataType = types.CloudEventsDataType{
+	Group:    "io.open-cluster-management.works",
+	Version:  "v1alpha1",
+	Resource: "manifests",
+}
+
+// SingleManifest represents the data in a cloudevent, it contains a single manifest.
+type SingleManifest struct {
```

Comment:
nit: do we need 'single', 'Manifest' is more simpler and straightforward?

Reply:
Use Manifest as the struct name